src/RouteMap.ts

```diff
@@ -43,6 +46,16 @@ export default class RouteMap {
         if (route.eventType === "API_GATEWAY") {
             route.resource = this.normalizePath(route.resource)
 
+            // on proxy we match event.resource to the url pattern
+            if(route.proxy){
```

Comment:
Wondering if it's possible to check `event.resource.includes("{proxy+}")`?

Instead of having to pass `proxy: true` into the router constructor??